### ISSUE TITLE
System.ArgumentException: Items collection cannot be modified when the DataSource property is set.

### ISSUE THREAD

**Issue by Nojwe** [phw198/OutlookGoogleCalendarSync#1510]
> Checklist:-
> * [X] [Upgrade](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Upgrading) to the latest release (alpha if available) ![latest release](https://img.shields.io/github/release/phw198/OutlookGoogleCalendarSync/all.svg)
> * [X] Reproduce the issue
> * [X] Complete the information below
> * [X] Provide your `OGcalsync.log` file (see [how-to](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file))

**OGCS Version**: v2.9.5.0
**Edition**: Happens on both Installed and Portable  
**Sync Direction**: G->O / O->G / O<->G

**Bug Description**  
Any time I try to add a profile, I get an error: 
System.ArgumentException: Items collection cannot be modified when the DataSource property is set. 
If I hit continue, the profile is still added as expected, but sync appears to not be working correctly. I get errors about orphaned outlook items and object reference not set to an instance of an object.

**Steps to Reproduce the Issue**  
1. Add a new profile
2. Click continue at the error
3. Try to sync

**Any other information**  
Log attached
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/9143535/OGcalsync.log)

**Comment by TotesFab** [comment]
I have the same issue when trying to add a Profile. At first I thought it was because I was using a profile name that had a special character "/", "-".  When I closed/restarted the app and deleted them (giving errors each time I switched), I closed/restarted again and used just a regular word, and the same error popped up.  Log attached

[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/9169089/OGcalsync.log)
